app/javascript/mastodon/features/ui/components/onboarding_modal.js

```diff
@@ -173,6 +175,13 @@ class OnboardingModal extends React.PureComponent {
     admin: ImmutablePropTypes.map
   };
 
+  constructor(props) {
+    super(props);
+
+    this.handleNext = this.handleNext.bind(this);
+    this.handleDot = this.handleDot.bind(this);
+  }
```

Comment:
Like above, I think this is required because it's called from `onClick` and then references `this`.